tests/Fixer/PhpUnit/PhpUnitShortWillReturnFixerTest.php

```diff
@@ -0,0 +1,255 @@
+                null,
+                ['returnValueMap' => false],
+            ],
+            'will return multiple occurrences with default configuration fixed all by default' => [
+                '<?php
+                $someMock->method(\'someMethod\')->willReturn(10);
+                $someMock->method(\'someMethod\')->willReturnSelf();
+                $someMock->method(\'someMethod\')->willReturnArgument(2);
+                $someMock->method(\'someMethod\')->willReturnCallback(\'str_rot13\');
+                $someMock->method(\'someMethod\')->willReturnMap([\'a\', \'b\', \'c\', \'d\']);
+                ',
+                '<?php
+                $someMock->method(\'someMethod\')->will($this->returnValue(10));
+                $someMock->method(\'someMethod\')->will($this->returnSelf());
+                $someMock->method(\'someMethod\')->will($this->returnArgument(2));
+                $someMock->method(\'someMethod\')->will($this->returnCallback(\'str_rot13\'));
+                $someMock->method(\'someMethod\')->will($this->returnValueMap([\'a\', \'b\', \'c\', \'d\']));
+                ',
+                [],
+            ],
+            'will return multiple occurrences with mixed configuration' => [
+                '<?php
+                $someMock->method(\'someMethod\')->will($this->returnValue(10));
+                $someMock->method(\'someMethod\')->willReturnSelf();
+                $someMock->method(\'someMethod\')->will($this->returnArgument(2));
+                $someMock->method(\'someMethod\')->willReturnCallback(\'str_rot13\');
+                $someMock->method(\'someMethod\')->will($this->returnValueMap([\'a\', \'b\', \'c\', \'d\']));
+                ',
+                '<?php
+                $someMock->method(\'someMethod\')->will($this->returnValue(10));
+                $someMock->method(\'someMethod\')->will($this->returnSelf());
+                $someMock->method(\'someMethod\')->will($this->returnArgument(2));
+                $someMock->method(\'someMethod\')->will($this->returnCallback(\'str_rot13\'));
+                $someMock->method(\'someMethod\')->will($this->returnValueMap([\'a\', \'b\', \'c\', \'d\']));
+                ',
+                [
+                    'returnValue' => false,
+                    'returnSelf' => true,
+                    'returnArgument' => false,
+                    'returnCallback' => true,
+                    'returnValueMap' => false,
+                ],
+            ],
+            'will return with multi lines and messy indents' => [
+                '<?php $someMock
+    ->method(\'someMethod\')
+        ->willReturn(10);',
+                '<?php $someMock
+    ->method(\'someMethod\')
+        ->will(
+            $this->returnValue(10)
+        );',
+            ],
+            'will return with multi lines, messy indents and comments inside' => [
+                '<?php $someMock
+    ->method(\'someMethod\')
+        ->willReturn(10)
+            // bar
```

Comment:
what happened to `// foo` ?

Reply:
That's the tricky part. I assumed that while this part is truncated, it should be removed along with comment. Otherwise there's problem where comment should be moved:
Should it be like:
```
<?php $someMock
    ->method('someMethod')
            // foo
        ->willReturn(10)
            // bar
        ;
```
or
```
<?php $someMock
    ->method('someMethod')
        ->willReturn
// foo
(10)
            // bar
        ;
```
or as I propose removed completly?

Another solution that comes to my mind is to skip all occurrences where comments appears.
Maybe some of you have better ideas how to solve it?

Reply:
no, don't skip cases with comments
no, don't remove comments

yeah, it's fine to move comment a little when needed :+1: